### PR TITLE
RHDEVDOCS-4016 Pipelines 1.7 RN revisions

### PR DESCRIPTION
**Pull request by sounix000**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: `enterprise-4.10`, `enterprise-4.11`
- **JIRA issues**: [RHDEVDOCS-4016 Revisions to Pipelines 1.7 RN](https://issues.redhat.com/browse/RHDEVDOCS-4016)
- **Preview pages**: 
  - [Installing OpenShift Pipelines](https://deploy-preview-45012--osdocs.netlify.app/openshift-enterprise/latest/cicd/pipelines/installing-pipelines.html) for the changes in installation channels.
  - [Release notes for Red Hat OpenShift Pipelines General Availability 1.7](https://deploy-preview-45012--osdocs.netlify.app/openshift-enterprise/latest/cicd/pipelines/op-release-notes.html#op-release-notes-1-7_op-release-notes)
- **SME review**: TBD       
- **Peer-review**: TBD

### CODE REVIEW

**Review by ppitonak — APPROVED** [review]

File: modules/op-installing-pipelines-operator-using-the-cli.adoc [hunk 1]
```diff
@@ -28,7 +28,7 @@ spec:
   source: redhat-operators <3>
   sourceNamespace: openshift-marketplace <4>
 ----
-<1> Specify the channel name from where you want to subscribe the Operator
+<1> Specify the channel name from where you want to subscribe the Operator. The `pipelines-<version>` channel is the default channel to install the {pipelines-title} Operator. For example, the default channel to install the {pipelines-title} Operator version `1.7` is `pipelines-1.7`. In addition, the `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator.
```
Comment (rolfedh) [inline modules/op-installing-pipelines-operator-using-the-cli.adoc:31]: The first sentence mentions "subscribe." The following ones mention "install."  Is that correct?

Comment (rolfedh) [reply modules/op-installing-pipelines-operator-using-the-cli.adoc:31]:
```suggestion
<1> The channel name of the Operator. Include the version number or `latest`. For example, `pipelines-1.7` or `pipelines-latest`. `latest` installs the most recent stable version of the {pipelines-shortname} Operator.
```
Parallelism. Most callouts use noun phrasing. Numbered callouts that look like numbered steps might cause confusion.

File: modules/op-release-notes-1-7.adoc [hunk 2]
```diff
@@ -15,6 +15,18 @@ In addition to the fixes and stability improvements, the following sections high
...
+* With this update, `pipelines-<version>` is the default channel to install the {pipelines-title} Operator. For example, the default channel to install the {pipelines-title} Operator version `1.7` is `pipelines-1.7`. Cluster administrators can also use the `latest` channel to install the most recent stable version of the Operator. 
++
+[NOTE]
+====
+The `preview` and `stable` channels will be deprecated and removed in a future release.
+====  
+
+* When you run a command in a user namespace, your container runs as `root` (user id `0`) but has user privileges on the host. With this update, to run pods in the user namespace, you must pass the annotations that link:https://cri-o.io/[CRI-O] expects.
+** To add these annotations for all users, run the `oc edit clustertask buildah` command and edit the `buildah` cluster task. 
+** To add the annotations to a specific namespace, export the cluster task as as task to that namespace.
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/op-release-notes-1-7.adoc:27]: Maybe
```suggestion
** To add the annotations to a specific namespace, export the cluster task to that namespace.
```

[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/op-release-notes-1-7.adoc:27]: Maybe use {pipelines-title} the first time and then switch to {pipelines-shortname}.

File: modules/op-release-notes-1-7.adoc [hunk 3]
```diff
@@ -15,6 +15,18 @@ In addition to the fixes and stability improvements, the following sections high
 [id="pipelines-new-features-1-7_{context}"]
 === Pipelines
 
+* With this update, `pipelines-<version>` is the default channel to install the {pipelines-title} Operator. For example, the default channel to install the {pipelines-title} Operator version `1.7` is `pipelines-1.7`. Cluster administrators can also use the `latest` channel to install the most recent stable version of the Operator.
```
Comment (rolfedh) [inline modules/op-release-notes-1-7.adoc:18]: Maybe use {pipelines-title} the first time and then switch to {pipelines-shortname}.

**Review by rolfedh — APPROVED** [review]
Minor suggestions. Pre-approved to expedite merging.

**Pull request merged**